ui/src/Elements/Message/README.md

```diff
@@ -0,0 +1,59 @@
+# Message
+
+This component represents a Kafka message, either produced to the broker or
+consumed from the broker. Produced and consumed messages are styled and
+structured differently and both can be supplied with interaction functions.
+
+## Properties
+
+Assuming use of the `Message` component directly, the named properties which are
+expected are as follows:
+
+- `className` - optional - add any specific styling classes to this component.
+This will be appended after the class which sets the type style, allowing
+modification of the styling if needed.
+- `usage` - required - the style and structure applied to this component depending
+  on the type of message. Must be either:
+  - `consumer` - for messages that are consumed from Kafka
+  - `producer` - for messages that are produced to Kafka
+- `isFirst` - optional - enables additional styling to indicate the first message.
```

Comment:
I didn't see an example of this in the storybook - the desired behaviour is a box with a small pill with the word 'Newest' in it, ala: https://github.com/matthew-chirgwin/kafka-java-vertx-starter/issues/55#issuecomment-645208448

Can a storybook entry be added, along with the string of text? I see that you do have a first classname defined for the producer/consumer in the component, but it isnt defined in the scss - is that deliberate?

Reply:
Sorry, missed this bit. Have updated.